python/lsst/rubintv/production/allSky.py

```diff
@@ -0,0 +1,468 @@
+# This file is part of rubintv_production.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import os
+import sys
+import time
+from time import sleep
+import logging
+import subprocess
+
+from lsst.summit.utils.utils import (dayObsIntToString,
+                                     getCurrentDayObs_int,
+                                     getCurrentDayObs_datetime,
+                                     )
+from lsst.rubintv.production import Uploader
+from lsst.rubintv.production.rubinTv import _dataIdToFilename
+
+try:
```

Comment:
`HAS_GOOGLE_STORAGE` isn't used anywhere.

Reply:
Yup, fair. I was expecting to need a different `Uploader` but turns out the existing one was fine. Gone.